luci-app-ssr-plus/luasrc/controller/shadowsocksr.lua

```diff
@@ -4,7 +4,7 @@ module("luci.controller.shadowsocksr", package.seeall)
 
 function index()
 	if not nixio.fs.access("/etc/config/shadowsocksr") then
-		return
+		call("act_reset")
```

Comment:
I would like to keep it as-is, to avoid loop issue in some cases.

Reply:
I don't think there will be a loop problem, to determine whether the configuration file exists, and create one if it does not exist.